src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -527,6 +527,12 @@ OutTask(OUTFUNC_ARGS)
 	WRITE_NODE_FIELD(rowValuesLists);
 	WRITE_BOOL_FIELD(partiallyLocalOrRemote);
 	WRITE_BOOL_FIELD(parametersInQueryStringResolved);
+	WRITE_NODE_FIELD(tupleDest);
```

Comment:
what does this do? 

could you try an `elog(NOTICE, "%s", nodeToString(someTask))`?

Reply:
Sure. This basically prints out the fields and values contained by some `Task`.
Here is an example output:
```
NOTICE:  {EXTENSIBLENODE :extnodename Task :taskType 1 :jobId 63501762561 :taskId 24 :taskQuery.queryType 1 :taskQuery.data.queryStringLazy SELECT\ a,\ b\ FROM\ test_102031\ test\ WHERE\ true :anchorDistributedTableId 0 :anchorShardId 102031 :taskPlacementList ({EXTENSIBLENODE :extnodename ShardPlacement :placementId 24 :shardId 102031 :shardLength 0 :shardState 1 :groupId 2 :nodeName localhost :nodePort 9702 :nodeId 2 :partitionMethod 104 :colocationGroupId 1 :representativeValue 939524096}) :dependentTaskList <> :partitionId 0 :upstreamTaskId 0 :shardInterval <> :assignmentConstrained false :replicationModel i :modifyWithSubquery false :relationShardList ({EXTENSIBLENODE :extnodename RelationShard :relationId 16819 :shardId 102031}) :relationRowLockList <> :rowValuesLists <> :partiallyLocalOrRemote false :parametersInQueryStringResolved true :tupleDest <> :queryCount 1 :totalReceivedTupleData 0 :fetchedExplainAnalyzePlacementIndex 0 :fetchedExplainAnalyzePlan Seq\ Scan\ on\ test_102031\ test\ \ \(cost=0.00..32.60\ rows=2260\ width=8\)\ \(actual\ time=0.001..0.001\ rows=0\ loops=1\)\
Planning\ Time:\ 0.044\ ms\
Execution\ Time:\ 0.009\ ms\
 :fetchedExplainAnalyzeExecutionDuration 0.01}
```

Reply:
Seems tupleDest was NULL here. I'm mainly wondering what happens if it's not NULL and whether it can be deparsed.